.github/workflows/coverage.yml

```diff
@@ -121,7 +121,7 @@ jobs:
       - run: |
           cat ./*profile.out | grep -v "mode: atomic" >> coverage.txt
         if: env.GIT_DIFF
-      - uses: codecov/codecov-action@v1.4.0
+      - uses: codecov/codecov-action@v1.4.1
```

Comment:
Is this related to the change?

Reply:
No, but otherwise:
![image](https://user-images.githubusercontent.com/7441766/115543307-859c9d00-a2a1-11eb-8a61-9af139fe3803.png)